attest/ake/src/lib.rs

```diff
@@ -50,37 +50,24 @@ mod test {
 
     #[test]
     fn ix_handshake() {
-        // Read an existing, valid quote
-        let data = include_str!("../test_data/ok_quote.txt");
-        let quote = Quote::from_base64(data.trim()).expect("Could not parse quote");
-
         // Create a new identity pubkey for our "enclave"
         let mut csprng = Hc128Rng::seed_from_u64(0);
         let identity = X25519Private::from_random(&mut csprng);
         let pubkey = X25519Public::from(&identity);
 
-        // Get the bytes from our quote
-        let mut quote_data = quote.to_x64_vec();
-
-        // Overwrite the cached quote's report_data contents with our pubkey
-        quote_data[368..400].copy_from_slice(pubkey.as_ref());
-
-        // Re-assemble a quote from the munged version
-        let quote = Quote::try_from(quote_data.as_ref())
-            .expect("Could not parse quote from modified bytes");
-
-        // Sign the forged quote with the sim client
-        let ra_client = Client::new("").expect("Could not create sim client");
-        let attestation_evidence = ra_client
-            .verify_quote(&quote, None)
-            .expect("Could not sign our bogus report");
+        let report_data = EnclaveReportDataContents::new([0x2au8; 16].into(), pubkey, [0x36u8; 32]);
```

Comment:
There was mention of trying to have a similar test to maintain the legacy EPID data here. The problem is that all of that logic is inside the `initiator.rs`. So can't really be tested here.
We would need to come up with a test in  `initiator.rs` using a fake responder